reference/src/functions/macros.md

```diff
@@ -490,6 +489,71 @@ let result = {
 };
 ```
 
+This can often be helpful when using "loop"-like macros to break early. For example in the
+`vector::any` macro
+
+```move
+public macro fun any<$T>($v: &vector<$T>, $f: |&$T| -> bool): bool {
```

Comment:
It may be a little confusing that the example for the "useful" case uses labelled return whereas the "original" one uses un-labelled return

Reply:
This one uses un-labelled return:
```
let result = apply!(|x| { if (x == 0) return 0; x + 1 }, 100);
```

And this one (which in text is mentioned as a more "useful" example of the one above uses labelled return:
```
v.do_ref!(|e| if ($f(e)) return 'any true);
```

It's minor, and it's probably OK to leave as is, but when I was reading this, I expected the first example and the second one to differ only in complexity rather than them using a subtly different control flow.

Reply:
I reworded to help this